docs/scenario-dashboard.md

```diff
@@ -0,0 +1,228 @@
+# Batch Scenario Manager
+
```

Comment:
First sentence should be something like:
"monoDrive's Batch Scenario Manager allows for easy batch processing of AV testing jobs."

Need to introduce the high level purpose before discussing the convenience of the UI around it